docs/execution-providers/Xnnpack-ExecutionProvider.md

```diff
@@ -0,0 +1,87 @@
+---
+title: XNNPACK
+description: Instructions to execute ONNX Runtime with the XNNPACK execution provider
+parent: Execution Providers
+nav_order: 16
+redirect_from: /docs/reference/execution-providers/Xnnpack-ExecutionProvider
```

Comment:
```suggestion

```

Reply:
We don't need this for new docs - the redirects are from legacy docs.

Reply:
Got it. Thanks